app/services/refresh_reporting_views.rb

```diff
@@ -42,12 +42,11 @@ def self.tz
     Rails.application.config.country[:time_zone]
   end
 
+  # Keep the views below in the order as defined, as they have dependencies on earlier matviews in the lists
   V1_REPORTING_VIEWS = %w[
     LatestBloodPressuresPerPatientPerMonth
     LatestBloodPressuresPerPatient
```

Comment:
The two views above are still used to build BloodPressuresPerFacilityPerDay, which is then used in my facilities. So we can't remove these until we have the right views for BPs per facility per day in the v2 pipeline.